LoopKit/DeviceManager/PumpManager.swift

```diff
@@ -182,6 +182,16 @@ public protocol PumpManager: DeviceManager {
     ///   - completion: A closure called after the command is complete
     ///   - result: A BasalRateSchedule or an error describing why the command failed
     func syncBasalRateSchedule(items scheduleItems: [RepeatingScheduleValue<Double>], completion: @escaping (_ result: Result<BasalRateSchedule, Error>) -> Void)
+
+    typealias SyncDeliveryLimits = (_ deliveryLimits: DeliveryLimits, _ completion: @escaping (_ result: Result<DeliveryLimits, Error>) -> Void) -> Void
```

Comment:
I think we should stop this pattern of creating types on the PumpManager api that are not used on the pumpmanager at all; this type is only used by the settings UI; it's part of the settings UI interface, and should live there.